middleware/passwordReset.js

```diff
@@ -0,0 +1,77 @@
+/**
+ *
+ * class to validate user reset password
+ * @class UserPassword
+ */
+class UserPassword {
+  /**
+ * Function validates forgot password with error messages
+ * @param {Object} req the request body
+ * @param {Object} res the response body
+ * @param {function} next a call to the nect function
+ * @returns {undefined}
+ */
+  static forgotPassword(req, res, next) {
+    const { email } = req.body.user;
+    if (email === undefined) {
+      return res.status(400)
+        .json({ message: 'Email field is required' });
```

Comment:
For consistency, your error response should be in the format: '{ errors: { message: 'error message' } }'

Reply:
Noted. Thanks for spotting the discrepancy in the error return statement